src/snakeoil/compression/__init__.py

```diff
@@ -156,6 +156,7 @@ def _unpack_cmd(self):
             for b in self.compress_binary:
                 try:
                     process.find_binary(b[0])
+                    # FIXME: This is a gnuism, needs gnu tar.
                     cmd += f' --use-compress-program="{" ".join(b)}"'
```

Comment:
`bsdtar` also supports this (at least for libarchive-3.6.1).  Thoughts on changing the binary lookup to `('gtar', 'bsdtar', 'tar')` ?